### ISSUE TITLE
How to use the trained model in tensorflow way

### ISSUE THREAD

**Issue by xiaoyaozzr** [tensorpack/tensorpack#807]
I just use the tensorpack trained one model.
When training ,  the inputs defined like this:
    def inputs(self):
        return **[tf.placeholder(self.image_dtype, [None, self.image_shape, self.image_shape, 3], 'input'),**
                tf.placeholder(tf.int32, [None], 'label') ]
when Testing(not using tensorpack way):
     tf.train.import_meta_graph('graph-123.meta', clear_devices=True)
     reader = tf.train.NewCheckpointReader('model/model-10000')

then 
logits_ = sess.run( ['logits'], feed_dict={'-----------': img})

the feed_dict here '-----------'  tensor is whitch one?  I tried 'input' and input is not right

**Comment by ppwwyyxx** [comment]
import_meta_graph should not be used to run a model. That's a very common tensorflow mistake: https://github.com/tensorpack/tensorpack/issues/790#issuecomment-396726606

http://tensorpack.readthedocs.io/tutorial/inference.html#inference-after-training
